docs/kyma/docs/036-inst-update.md

```diff
@@ -0,0 +1,89 @@
+---
+title: Update Kyma
+type: Installation
+---
+
+This guide describes how to update Kyma deployed locally or on a cluster.
+
+## Prerequisites
+
+- [Docker](https://www.docker.com/)
+- Access to a Docker Registry - only for cluster installation
+
+## Overview
+
+Kyma consists of multiple components, installed as [Helm](https://github.com/helm/helm/tree/master/docs) releases.
+
+Update of an existing deployment can include:
+- changes in charts
+- changes in overrides
+- adding new releases
+
+The update procedure consists of three main steps:
+- Prepare the update
+- Update the Kyma Installer
+- Trigger the update process
+
+> **NOTE:** In case of dependency conflicts or major changes between components versions, some updates may not be possible.
+
+> **NOTE:** Currently Kyma doesn't support removing components as a part of the update process.
+
+
+## Prepare the update
+
+- If you update and existing component, make all required changes to the Helm charts of the component located in the in the `[resources](https://github.com/kyma-project/kyma/tree/master/resources)` directory.
```

Comment:
you cannot use the code ` sings like this because your sentence cannot be rendered in a proper way, see
<img width="900" alt="screen shot 2019-01-15 at 14 55 18" src="https://user-images.githubusercontent.com/17568639/51184845-c7298b00-18d5-11e9-8249-402235374f62.png">

if you want to have the **resources** in a block code then you need to move your ` sings inside the brackets, like this one:
```
[`resources`](https://github.com/kyma-project/kyma/tree/master/resources)  directory.
```

this will produce such output 
[`resources`](https://github.com/kyma-project/kyma/tree/master/resources) directory.

Reply:
Thanks, Fixed.